### ISSUE TITLE
Table not working? 

### ISSUE THREAD

**Issue by ubaldot** [ManimCommunity/manim#3453]
It looks like that `Table` class is not working, see below.

## Example

```py
import manim as mn

class TableExamples(mn.Scene):
    def construct(self):
        t0 = mn.Table([["This", "is a"], ["simple", "Table in \n Manim."]])
        self.add(t0)
```

## Comamnd used
`manim myfile.py TableExamples -pql --fps 30 --disable_caching -v DEBUG`

## Traceback
```
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ /opt/homebrew/Caskroom/miniconda/base/envs/manim_ce/lib/python3.11/site-packages/manim/cli/rende │
│ r/commands.py:115 in render                                                                      │
│                                                                                                  │
│   112 │   │   │   try:                                                                           │
│   113 │   │   │   │   with tempconfig({}):                                                       │
│   114 │   │   │   │   │   scene = SceneClass()                                                   │
│ ❱ 115 │   │   │   │   │   scene.render()                                                         │
│   116 │   │   │   except Exception:                                                              │
│   117 │   │   │   │   error_console.print_exception()                                            │
│   118 │   │   │   │   sys.exit(1)                                                                │
│                                                                                                  │
│ /opt/homebrew/Caskroom/miniconda/base/envs/manim_ce/lib/python3.11/site-packages/manim/scene/sce │
│ ne.py:223 in render                                                                              │
│                                                                                                  │
│    220 │   │   """                                                                               │
│    221 │   │   self.setup()                                                                      │
│    222 │   │   try:                                                                              │
│ ❱  223 │   │   │   self.construct()                                                              │
│    224 │   │   except EndSceneEarlyException:                                                    │
│    225 │   │   │   pass                                                                          │
│    226 │   │   except RerunSceneException as e:                                                  │
│                                                                                                  │
│ /Users/ubaldot/Documents/YouTube/ControlTheoryInPractice/github_ctip/07_Norms/discord.py:40 in   │
│ construct                                                                                        │
│                                                                                                  │
│   37                                                                                             │
│   38 class TableExamples(mn.Scene):                                                              │
│   39 │   def construct(self):                                                                    │
│ ❱ 40 │   │   t0 = mn.Table([["This", "is a"], ["simple", "Table in \n Manim."]])                 │
│   41 │   │                                                                                       │
│   42 │   │   self.add(t0)                                                                        │
│   43                                                                                             │
│                                                                                                  │
│ /opt/homebrew/Caskroom/miniconda/base/envs/manim_ce/lib/python3.11/site-packages/manim/mobject/t │
│ able.py:238 in __init__                                                                          │
│                                                                                                  │
│    235 │   │   │   │   raise ValueError("Not all rows in table have the same length.")           │
│    236 │   │                                                                                     │
│    237 │   │   super().__init__(**kwargs)                                                        │
│ ❱  238 │   │   mob_table = self._table_to_mob_table(table)                                       │
│    239 │   │   self.elements_without_labels = VGroup(*it.chain(*mob_table))                      │
│    240 │   │   mob_table = self._add_labels(mob_table)                                           │
│    241 │   │   self._organize_mob_table(mob_table)                                               │
│                                                                                                  │
│ /opt/homebrew/Caskroom/miniconda/base/envs/manim_ce/lib/python3.11/site-packages/manim/mobject/t │
│ able.py:274 in _table_to_mob_table                                                               │
│                                                                                                  │
│    271 │   │   List                                                                              │
│    272 │   │   │   List of :class:`~.VMobject` from the entries of ``table``.                    │
│    273 │   │   """                                                                               │
│ ❱  274 │   │   return [                                                                          │
│    275 │   │   │   [                                                                             │
│    276 │   │   │   │   self.element_to_mobject(item, **self.element_to_mobject_config)           │
│    277 │   │   │   │   for item in row                                                           │
│                                                                                                  │
│ /opt/homebrew/Caskroom/miniconda/base/envs/manim_ce/lib/python3.11/site-packages/manim/mobject/t │
│ able.py:275 in <listcomp>                                                                        │
│                                                                                                  │
│    272 │   │   │   List of :class:`~.VMobject` from the entries of ``table``.                    │
│    273 │   │   """                                                                               │
│    274 │   │   return [                                                                          │
│ ❱  275 │   │   │   [                                                                             │
│    276 │   │   │   │   self.element_to_mobject(item, **self.element_to_mobject_config)           │
│    277 │   │   │   │   for item in row                                                           │
│    278 │   │   │   ]                                                                             │
│                                                                                                  │
│ /opt/homebrew/Caskroom/miniconda/base/envs/manim_ce/lib/python3.11/site-packages/manim/mobject/t │
│ able.py:276 in <listcomp>                                                                        │
│                                                                                                  │
│    273 │   │   """                                                                               │
│    274 │   │   return [                                                                          │
│    275 │   │   │   [                                                                             │
│ ❱  276 │   │   │   │   self.element_to_mobject(item, **self.element_to_mobject_config)           │
│    277 │   │   │   │   for item in row                                                           │
│    278 │   │   │   ]                                                                             │
│    279 │   │   │   for row in table                                                              │
│                                                                                                  │
│ /opt/homebrew/Caskroom/miniconda/base/envs/manim_ce/lib/python3.11/site-packages/manim/mobject/t │
│ ext/text_mobject.py:160 in __init__                                                              │
│                                                                                                  │
│    157 │   │   super().__init__()                                                                │
│    158 │   │                                                                                     │
│    159 │   │   lines_str = "\n".join(list(text))                                                 │
│ ❱  160 │   │   self.lines_text = Text(lines_str, line_spacing=line_spacing, **kwargs)            │
│    161 │   │   lines_str_list = lines_str.split("\n")                                            │
│    162 │   │   self.chars = self._gen_chars(lines_str_list)                                      │
│    163                                                                                           │
│                                                                                                  │
│ /opt/homebrew/Caskroom/miniconda/base/envs/manim_ce/lib/python3.11/site-packages/manim/mobject/t │
│ ext/text_mobject.py:486 in __init__                                                              │
│                                                                                                  │
│    483 │   │   │   self.line_spacing = self._font_size + self._font_size * self.line_spacing     │
│    484 │   │                                                                                     │
│    485 │   │   color = Color(color) if color else VMobject().color                               │
│ ❱  486 │   │   file_name = self._text2svg(color)                                                 │
│    487 │   │   PangoUtils.remove_last_M(file_name)                                               │
│    488 │   │   super().__init__(                                                                 │
│    489 │   │   │   file_name,                                                                    │
│                                                                                                  │
│ /opt/homebrew/Caskroom/miniconda/base/envs/manim_ce/lib/python3.11/site-packages/manim/mobject/t │
│ ext/text_mobject.py:814 in _text2svg                                                             │
│                                                                                                  │
│    811 │   │   │   width = config["pixel_width"]                                                 │
│    812 │   │   │   height = config["pixel_height"]                                               │
│    813 │   │   │                                                                                 │
│ ❱  814 │   │   │   svg_file = manimpango.text2svg(                                               │
│    815 │   │   │   │   settings,                                                                 │
│    816 │   │   │   │   size,                                                                     │
│    817 │   │   │   │   line_spacing,                                                             │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯

```
## My system
I am running on Manim CE 17.3 on macos Sonoma, python 3.11.4 installed through Miniconda.

**Comment by uwezi** [comment]
Can you post the full error message - the code renders just fine on my computer.

Do you possibly have an issue with the Pango library on your system? Does the following code work for you? If not then that's your problem, since `Table()` uses the `Text()` object.

```py
import manim as mn

class textPango(Scene):
    def construct(self):
        text = Text("Hello world")
        self.add(text)
```

**Comment by ubaldot** [comment]
There we go! 
Yes, there is a Pango problem. 

Everything happened after I run `conda update manim` thinking that I was upgrading to v18.0 but from v17.3 (that I have installed) it eventually get updated/downgraded to a XYZ version (I have no idea where and I realized only after that v18.0 was not yet available on `conda`). 

Anyway, any suggestion on how to fix it "on the fly" or shall I throw away this virtual environment and create another one from scratch (sooo annoying!!!)?
